package/iframe/js/main.js

```diff
@@ -65,7 +65,7 @@ function buildQS(profile) {
     }
 
     if (profile) {
-        qs.push('pro=' + profile);
+        qs.push('pro=' + encodeURIComponent(profile));
```

Comment:
One of the things I want to check. We didn't have issues with the profile before, but now it can contain `'='`, `'/'`, etc. Need to make sure this works transparently.